src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const ROBOT_VALUES = Object.values(robot);
+
+  function findDuplicates(arry) {
+    return arry.filter((item, index) => arry.indexOf(item) !== index);
+  };
+
+  const DUP_VALUES = findDuplicates(ROBOT_VALUES);
+
+  if (DUP_VALUES.length >= 1) {
```

Comment:
Instead of checking if 'DUP_VALUES.length >= 1', you can use 'if (DUP_VALUES.length)' which will evaluate to 'true' if there are any duplicates.